### PR TITLE
Config: Implement court config setter

### PR DESCRIPTION
**Pull request by facuspagnuolo**

### CODE REVIEW

File: data/input/court.usability.js [hunk 1]
```diff
@@ -29,11 +29,11 @@ module.exports = {
   },
```
Comment (sohkai) [inline data/input/court.usability.js:29]: Now I'm wondering... maybe we just completely remove `usability`—opening up an open Rinkeby test environment is something that we can re-evaluate after launching Agreements.

Could we just maintain `staging` as the "fast test" environment?

File: src/commands/set-config.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+const command = 'set-config'
+const describe = 'Change Court config'
+
+const builder = {
+  input: { alias: 'i', describe: 'Court config JS file', type: 'string', default: './data/input/court-addresses' },
+}
+
+const handlerAsync = async (environment, { network, input }) => {
+  const config = require(path.resolve(process.cwd(), input))[network]
```
Comment (sohkai) [inline src/commands/set-config.js:12]: Not sure if we do this for other commands, but I think it'd be nice to log which config file is being used :)

File: data/input/court.usability.js (hunk 1 above)
Comment (facuspagnuolo) [reply data/input/court.usability.js:29]: Yea, I think having two of these is more than enough, one mimicking mainnet and the other one faster for the daily basis

File: src/commands/set-config.js (hunk 2 above)
Comment (facuspagnuolo) [reply src/commands/set-config.js:12]: we don't but I agree it would be nice :)

**Pull request merged**